client/src/app/shared/components/os-sort-filter-bar/os-sort-filter-bar.component.html

```diff
@@ -0,0 +1,35 @@
+<div class='custom-table-header on-transition-fade os-sort-filter-bar'>
+    <button mat-button *ngIf="filters.length" (click)="filterMenu.opened ? filterMenu.close() : filterMenu.open()">
+        <span translate>FILTER</span>
+    </button>
+    <button mat-button *ngIf="vp.isMobile && sorting && sorting.options.length" (click)="openSortDropDown()">
+        <span translate>SORT</span>
+    </button>
+    <button mat-button *ngIf="!vp.isMobile && sorting && sorting.options.length" [matMenuTriggerFor]="menu">
+        <span translate>SORT</span>
+    </button>
+    <input matInput *ngIf="isSearchBar" (keyup)="applySearch($event, $event.target.value)" osAutofocus
+        placeholder="{{translate.instant('Search')}}" [ngClass]= "vp.isMobile? 'vp':'fullscreen'">
+    <span>
+        <mat-icon (click)="isSearchBar = !isSearchBar">search</mat-icon>
+    </span>
+</div>
+<mat-drawer #filterMenu mode="push" position="end">
+    <div class="custom-table-header filter-menu" (click)="this.filterMenu.toggle()">
+        <span translate>Filter options</span>
+        <div class=spacer></div>
+        <mat-icon>keyboard_arrow_right</mat-icon>
+    </div>
+    <os-filter-menu [filterOptions]="filters" (activeFilters)="newFilters($event)" (dismissed)="this.filterMenu.close()">
+        </os-filter-menu>
+</mat-drawer>
+
+<mat-menu #menu>
+    <div *ngIf="sorting && sorting.options.length">
+        <mat-list-item *ngFor="let option of sorting.options" (click)="sortingChange(option)">
+            <button mat-menu-item>
+            <mat-icon>{{ getIcon(option) }}</mat-icon>
+            <span translate> {{ getLabel(option) }}</span></button>
```

Comment:
Whitespace between `>` and `{{`
If you want this, for some reason, you might add a protected whitespace before the span